src/routes/docs/products/databases/csv-imports/+page.markdoc

```diff
@@ -35,14 +55,71 @@ mz7lq3dp5c,2025-08-11T09:15:00.000Z,2025-08-11T10:00:00.000Z,The Fellowship of t
 x0v4p8ncq2,2025-08-12T08:00:00.000Z,2025-08-12T08:30:00.000Z,To Kill a Mockingbird,Harper Lee,1960,false
 ```
 
-{% info title="Note" %}
-All required columns must be present in the CSV. Each row is validated before being imported.
-{% /info %}
-
 {% info title="Timestamps format" %}
 `$createdAt` and `$updatedAt` must be valid ISO 8601 date-time strings, for example: `2025-08-10T12:34:56.000Z`.
 {% /info %}
 
+# Add arrays {% #add-arrays %}
+
+You can also include data for array columns in your CSV file. For any column configured as an array, you can include a comma-separated list of values within double quotes (`"one,two"`).
+
+An example of a valid CSV file with an array column:
+
+```text
+$id,title,author,year,available,categories
+f3k91x8b2q,Harry Potter and the Sorcerer's Stone,J.K. Rowling,1997,true,"fiction,fantasy"
+mz7lq3dp5c,The Fellowship of the Ring,J.R.R. Tolkien,1954,true,"fiction,fantasy"
+x0v4p8ncq2,To Kill a Mockingbird,Harper Lee,1960,false,"fiction,nonfiction"
+```
+
+# Relationships between tables {% #relationships-between-tables %}
+
+If you want to create relationships between rows in different tables, you can provide the `$id` of the related row in the foreign table.
```

Comment:
lets use related table and not foreign. we've never used that terminology before.